sdk/src/types/api/core/response.rs

```diff
@@ -56,11 +69,11 @@ pub struct StatusResponse {
     pub confirmed_tangle_time: u64,
     #[serde(with = "crate::utils::serde::string")]
     pub relative_confirmed_tangle_time: u64,
-    pub latest_committed_slot: SlotIndex,
+    pub latest_commitment_id: SlotCommitmentId,
     pub latest_finalized_slot: SlotIndex,
+    pub latest_accepted_block_slot: BlockId,
+    pub latest_confirmed_block_slot: BlockId,
```

Comment:
These two should be `SlotIndex`

Reply:
Not sure how I managed to get that wrong